versions/1/data_stream/manifest.spec.yml

```diff
@@ -87,6 +87,14 @@ spec:
       - beta
       examples:
       - beta
+    ilm_policy:
+      description: The name of an existing ILM (Index Lifecycle Management) policy
```

Comment:
Is it a part of the package (is it an ordinary file)?

Reply:
yes it is under elastic/ilm

Reply:
I think what Marcin is getting at here: Do we expect here that it only references a file which is under `data_stream/*/elastic/ilm_policy/*` or can it be any name? This is relevant for validation.

I think we also need to have a conversation around how we name these ILM policies which are specific to a data_stream. Suggestions?

Last, will the same ILM policy apply for all data_streams of this type?

Reply:
> I think what Marcin is getting at here: Do we expect here that it only references a file which is under data_stream/*/elastic/ilm_policy/* or can it be any name? This is relevant for validation.

I could see it both ways. I'm thinking it's better to start with the more strict validation (the ilm_policy is provided in the same data stream definition) and relax it later if we see the need for multiple to reference the same file. But I don't know if that adds to the complexity.

> Last, will the same ILM policy apply for all data_streams of this type?

In this particular use case, I don't really forsee having more than the `default` namespace, so for simplicity, I think it's ok to apply it to all to start with?

Reply:
++

An other interesting discussion we had in a meeting is that the specific ILM policies for a data_stream should be overwritten on upgrade in contrast to the ILM policies we load on the global level.